heaps/heap_sort.py

```diff
@@ -1,8 +1,18 @@
-
+from heaps.min_heap import MinHeap
 
 def heap_sort(list):
     """ This method uses a heap to sort an array.
-        Time Complexity:  ?
-        Space Complexity: ?
+        Time Complexity:  O(n)
+        Space Complexity: O(1)
     """
```

Comment:
👍  You add n elements to a heap and each add takes O(log n) times.  So the time complexity is O(n log n).

You also add each element to the heap, n elements, so the space complexity is O(n)